post/tests/test105/source.c:13: Handling error in variable main#retval, which must contain one of the following error codes: TENTATIVE_EIO* 
post/tests/test105/source.c:12:"retval" may have an unchecked error
post/tests/test105/source.c:11:"retval" may have an unchecked error
post/tests/test105/source.c:9:"retval" may have an unchecked error
post/tests/test105/source.c:8:"retval" receives an error from "TENTATIVE_EIO"

post/tests/test105/source.c:13: Handling error in variable main#retval, which must contain one of the following error codes: TENTATIVE_EIO* 
post/tests/test105/source.c:8:"retval" receives an error from "TENTATIVE_EIO"
====
post/tests/test105/source.c:17: Handling error in variable main#retval, which may contain one of the following error codes: TENTATIVE_ENXIO* TENTATIVE_EIO 
post/tests/test105/source.c:13:"retval" receives an error from "err"
post/tests/test105/source.c:13:"err" may have an unchecked error
post/tests/test105/source.c:12:"err" may have an unchecked error
post/tests/test105/source.c:11:"err" may have an unchecked error
post/tests/test105/source.c:9:"err" receives an error from "TENTATIVE_ENXIO"

post/tests/test105/source.c:17: Handling error in variable main#retval, which may contain one of the following error codes: TENTATIVE_ENXIO* TENTATIVE_EIO 
post/tests/test105/source.c:13:"retval" receives an error from "err"
post/tests/test105/source.c:9:"err" receives an error from "TENTATIVE_ENXIO"
====
post/tests/test105/source.c:18: Handling error in variable main#err, which may contain one of the following error codes: TENTATIVE_ENXIO* 
post/tests/test105/source.c:17:"err" may have an unchecked error
post/tests/test105/source.c:13:"err" may have an unchecked error
post/tests/test105/source.c:12:"err" may have an unchecked error
post/tests/test105/source.c:11:"err" may have an unchecked error
post/tests/test105/source.c:9:"err" receives an error from "TENTATIVE_ENXIO"

post/tests/test105/source.c:18: Handling error in variable main#err, which may contain one of the following error codes: TENTATIVE_ENXIO* 
post/tests/test105/source.c:9:"err" receives an error from "TENTATIVE_ENXIO"
====
